ckanext/canada/templates/public/admin/jobs.html

```diff
@@ -0,0 +1,60 @@
+{% extends "admin/base.html" %}
+
+{% block primary_content_inner %}
+<div class="col-sm-12">
+  {% if job_list %}
+    {% if warning %}
+      <div class="module-alert alert alert-warning">
+        <p>{{ _('The first job in the queue has not run in the past 18 minutes, you may want to confirm that the background worker is running.') }}</p>
+      </div>
+    {% endif %}
+    <div class="text-left text-info mrgn-bttm-md canada-job-list-title">
+      <p>
+        <strong>{{ total_job_count ~ _(" job(s) in queue.") }}</strong>
+        {% if total_job_count > job_list|length %}
+          <br/>
+          <strong><small>{{ _("Showing ") ~ job_list|length ~ "/" ~ total_job_count }}</small></strong>
+        {% endif %}
+      </p>
+    </div>
+    <ul class="canada-job-list activity">
+      {% for job in job_list %}
+        <li class="item">
+          <i class="fa icon {{ job.get('icon') }}"></i>
+          <p>
+            <span class="canada-job-item-title">{{ job.get('type') }}</span>
+            <br/>
+            <span class="date">
+              {{ job.get('since_time') }} ({{ job.get('created_human_readable') }}) |
+              {% if job.get('info') %}
+                <a href="{{ job.get('info', {}).get('url') }}">{{ h.get_translated(job.get('info'), 'name') }}</a>
+              {% else %}
+                <span>{{ _('Unknown') }}</span>
+              {% endif %}
+              | <a href="{{ h.url_for('api.action', logic_function='job_show', id=job.get('id')) }}">{{ job.get('id') }}</a>
+            </span>
+          </p>
+        </li>
+      {% endfor %}
+    </ul>
+  {% else %}
+    <div class="module-alert alert alert-info">
+      <p>{{ _("There are currently no queued or running jobs.") }}</p>
+    </div>
+  {% endif %}
+</div>
+{% endblock %}
+
+{% block secondary_content %}
+  <section class="panel panel-info">
+    <header class="panel-heading">
+        <h3 class="panel-title">
+            <i class="fa fa-info-circle"></i>
+            {{ _('Job Queue') }}
+        </h3>
+    </header>
+    <div class="panel-body">
+      <p>{{ _("View the current job queue and running jobs.") }}</p>
+    </div>
+  </section>
+{% endblock %}
```

Comment:
an admin page for listing jobs would be a great upstream feature

Reply:
will make a ticket in the backlog for it